models/intermediate/users.sql

```diff
@@ -69,6 +69,7 @@ FROM {{ var('source_db') }}.users AS u
   	LEFT JOIN {{ var('source_db') }}.registry_orgs AS ro ON o.id = ro.org_id
     LEFT JOIN {{ var('source_db') }}.registry_orgs AS ron ON o.name COLLATE utf8mb3_unicode_ci = ron.name
   LEFT JOIN {{ var('source_db') }}.identifiers orc ON orc.identifiable_type = 'User' AND orc.identifiable_id = u.id AND orc.identifier_scheme_id = 1
-  LEFT JOIN {{ var('source_db') }}.identifiers sso ON sso.identifiable_type = 'User' AND sso.identifiable_id = u.id AND sso.identifier_scheme_id = 2;
+  LEFT JOIN {{ var('source_db') }}.identifiers sso ON sso.identifiable_type = 'User' AND sso.identifiable_id = u.id AND sso.identifier_scheme_id = 2
+GROUP BY u.id, u.firstname, u.surname, u.email, u.created_at, u.updated_at, u.accept_terms, u.active, u.last_sign_in_at, l.abbreviation, u.encrypted_password, u.org_id;
```

Comment:
Could probably have been more elegant here by using some `WITH` queries instead. 

I may do so in the future since there are still duplicates leaking through but this is working well enough for now. I left a `TODO` comment in the FinalMigration doc and will create a separate ticket